### ISSUE TITLE
KeyError: 'regression' after model.train_model(train_df)

### ISSUE THREAD

**Issue by phrenol** [ThilinaRajapakse/simpletransformers#145]
Hi, Thanks for this awesome lib.

I'm playing around with it. Currently I'm facing an issue. 

```
model = MultiLabelClassificationModel('roberta', 
                                      'roberta-base', 
                                      num_labels=10, 
                                       args={
                                          'output_dir': 'outputs/',
                                          'cache_dir': 'cache/',
                                          'fp16': True,
                                          'fp16_opt_level': 'O1',
}
)
model.train_model(train_df)

```
causes

![image](https://user-images.githubusercontent.com/12728649/72371268-4d082e00-3704-11ea-83e1-ac35dee57d04.png)

**Comment by phrenol** [comment]
hmmm, adding 
```'regression': False,```
to the args fixes the issue

**Comment by ThilinaRajapakse** [comment]
The issue was due to the `regression` key being added to the ClassificationModel but not to the MultiLabelClassificationModel (which inherits from ClassificationModel).

I pushed a fix with 0.18.1 which should avoid checking for the `regression` key in multi-label tasks.

**Comment by phrenol** [comment]
Thanks for the quick fix. This lib is awesome.
